webapp/src/css/inbox.less

```
@@ -1258,11 +1258,29 @@ mm-search-bar {
     .fa {
       font-size: @font-extra-large;
       color: @filter-icon-color;
+      vertical-align: top;
+      &.fa-qrcode {
+        font-size: @font-extra-extra-large;
+      }
+
+      &.fa-sliders {
+        vertical-align: baseline;
+      }
+
+      &:not(:first-child) {
+        margin-left: 10px;
+      }
     }
 
     &.disabled .search-bar-left-icon .fa {
       color: @inactive-color;
     }
+
+    #barcode-scanner-input {
+      visibility: hidden;
+      position: absolute;
+      left: -1000px;
```

```suggestion
      display: none;
```
if the element should not take any space on the page and is not going to be animated, it's better to use just `display: none`  
an element with `visibility: hidden` still gets rendered by the browser and occupies space on the page

---

Nice!